models/repository/repository.go

```diff
@@ -45,3 +60,16 @@ func (u *RepositoryStruct) Save() bool {
 	u.ID, _ = result.LastInsertId()
 	return true
 }
+
+func (u *RepositoryStruct) Authenticate() bool {
```

Comment:
__[golint]__
* [golint] exported method RepositoryStruct.Authenticate should have comment or be unexported